x-pack/plugins/console_extensions/spec/overrides/xpack.ml.get_categories.json

```diff
@@ -0,0 +1,10 @@
+{
+  "xpack.ml.get_categories": {
+    "data_autocomplete_rules": {
```

Comment:
FWIW, I can only get the autocompleter to suggest these for `_xpack/ml/anomaly_detectors/{job_id}/results/categories/{category_id}`, but not `_xpack/ml/anomaly_detectors/{job_id}/results/categories`

Reply:
Good catch, generated file had that path end with a slash, fixed it in overrides.